.github/workflows/build.yml

```diff
@@ -140,6 +143,8 @@ jobs:
 
       - name: 'Prepare PR comment'
         if: always()
+        env:
+          SANITY_ERROR_MESSAGE: ${{ steps.sanity_check_pr_content.outputs.sanity-error-message }}
```

Comment:
We set a variable here for a sanity check error but if there is a sanity check error we would never get here because of the  "Reflect on Sanity Check PR Content" step which exits on sanity check failure well before we get here.

Reply:
There is an `if: always()` condition (line. 145), so this is going to run irrespective of any other previous job fails or not.

Reply:
So "exit 1" doesn't exit, perfectly obvious! Got it.